presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -166,6 +173,11 @@ public JoinNode join(JoinNode.Type type, PlanNode left, PlanNode right, List<Joi
         return new JoinNode(idAllocator.getNextId(), type, left, right, criteria, outputSymbols, filter, Optional.empty(), Optional.empty(), Optional.empty());
     }
 
+    public JoinGraphNode joinGraph(List<PlanNode> sources, List<JoinNode.EquiJoinClause> criteria, List<Expression> filters)
```

Comment:
It doesn't get used until here (that one needs the matcher, but the planBuilder input is not a join graph node).